packages/text/src/TextBase.js

```diff
@@ -15,11 +15,7 @@ const TextBase = styled.span`
     if (props.dashed) return '1px dashed'
     return 'none'
   }};
-  cursor: ${(props) => {
-    if (props.cursor) return props.cursor
-    if (props.a) return 'pointer'
-    return 'auto'
-  }};
+  ${({ a, cursor }) => (a || cursor) && `cursor: ${(cursor || 'pointer')};`}
```

Comment:
nice

Reply:
While I'd like to take credit, a lot of this is the linter's doing.

Reply:
haha, nice